SonarQube.TeamBuild.PreProcessor/Interfaces/IAnalyzerProvider.cs

```diff
@@ -19,6 +20,6 @@ public interface IAnalyzerProvider
         /// </summary>
         /// <param name="projectKey">Identifier for the project being analyzed</param>
         /// <returns>The settings required to configure the build for Roslyn analysis</returns>
-        AnalyzerSettings SetupAnalyzers(ISonarQubeServer server, TeamBuildSettings settings, string projectKey);
+        AnalyzerSettings SetupAnalyzers(ISonarQubeServer server, TeamBuildSettings settings, string projectKey, string projectBranch);
```

Comment:
From offline discussion - [here](https://github.com/trevmex/EnvJasmine/blob/master/sonar-project.properties) is an example of a sonar-project.properties file with sonar.branch.